clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/PartitionReplicaIngestionContext.java

```diff
@@ -61,22 +58,14 @@ public WorkloadType getWorkloadType() {
     return workloadType;
   }
 
-  public static WorkloadType getWorkloadType(PubSubTopic versionTopic, Store store) {
-    checkVersionTopicStoreOrThrow(versionTopic, store);
-    int versionNumber = Version.parseVersionFromKafkaTopicName(versionTopic.getName());
-    Version version = store.getVersion(versionNumber);
-    if (version == null) {
-      return WorkloadType.NON_AA_OR_WRITE_COMPUTE;
-    }
-    if (store.isWriteComputationEnabled() || version.isActiveActiveReplicationEnabled()) {
+  public static WorkloadType getWorkloadType(boolean isActiveActiveReplicationEnabled, Store store) {
```

Comment:
This is not sufficient condition right? If even batch portion of data will land in  `AA_OR_WRITE_COMPUTE` pool

Reply:
This is handled during subscription. So no change required